refinedstorage2-platform-common/src/main/java/com/refinedmods/refinedstorage2/platform/common/block/entity/AbstractInternalNetworkNodeContainerBlockEntity.java

```diff
@@ -114,11 +115,35 @@ public boolean canPerformOutgoingConnection(final Direction direction) {
 
     @Override
     public boolean canAcceptIncomingConnection(final Direction direction, final BlockState other) {
+        if (!colorsAllowConnecting(other)) {
```

Comment:
Could we write this as a strategy, so addons can inject their own custom logic in here?